.github/scripts/generate_binary_build_matrix.py

```diff
@@ -45,7 +45,7 @@
         "nvidia-cuda-nvrtc-cu11==11.8.89; platform_system == 'Linux' and platform_machine == 'x86_64' | "  # noqa: B950
         "nvidia-cuda-runtime-cu11==11.8.89; platform_system == 'Linux' and platform_machine == 'x86_64' | "
         "nvidia-cuda-cupti-cu11==11.8.87; platform_system == 'Linux' and platform_machine == 'x86_64' | "
-        "nvidia-cudnn-cu11==9.1.0.70; platform_system == 'Linux' and platform_machine == 'x86_64' | "
```

Comment:
Let's not change this.

Reply:
Why not?

Reply:
Oh, it is not changed. Not sure why I left this comment... Closing.